props/saber_fett263_buttons.h

```diff
@@ -293,166 +664,3131 @@ class SaberFett263Buttons : public virtual PropBase {
+    EDIT_COLORS = 2,
+    EDIT_FONT = 3,
+    EDIT_TRACK = 4,
+    EDIT_SETTINGS = 5,
+    COPY_PRESET = 6,
+    DELETE_PRESET = 7,
+    EXIT_MENU = 8,
+  };
+  
+  enum ColorSubMenu {
+    EDIT_EFFECT_COLOR = 1,
+#if NUM_BLADES == 1
+    RESET_COLORS = 2,
+#else
+    COPY_COLORS = 2,
+    RESET_COLORS = 3,
+#endif
+  };
+  enum ColorOptions {
+    COLOR_LIST_MODE = 0,
+    COLOR_HUE_MODE = 1,
+    WHITE_MODE = 2,
+    BLACK_MODE = 3,
+  }; 
+   
+  enum SettingsMenu {
+    EDIT_VOLUME = 1,
+    EDIT_STYLE_SETTINGS = 2,
+    EDIT_CONTROL_SETTINGS = 3,
+    EDIT_CLASH_THRESHOLD = 4,
+    EDIT_BLADE_LENGTH = 5,
+    EDIT_BRIGHTNESS = 6,
+  };
+   
+  enum GestureControls {
+    GESTURE_SWINGON_IGNITION = 1,
+    GESTURE_SWINGON_SPEED = 2,
+    GESTURE_TWIST_IGNITION = 3,
+    GESTURE_THRUST_IGNITION = 4,
+    GESTURE_STAB_IGNITION = 5,
+    GESTURE_FORCE_PUSH = 6,
+    GESTURE_FORCE_PUSH_LENGTH = 7,
+    GESTURE_TWIST_RETRACTION = 8,
+    GESTURE_LOCKUP_DELAY = 9,
+    GESTURE_CLASH_DETECTION = 10,
+    GESTURE_POWER_LOCK = 11,
+    GESTURE_MAX_CLASH = 12,
+  };
+
+// Gesture Control Boolean Menu
+  void EnterBooleanMenu(MenuType t, bool current_value) {
+    menu_type_ = t;
+    choice_ = current_value;
+    if (current_value) {
+      PlayMenuSound("mtrue.wav");
+    } else {
+      PlayMenuSound("mfalse.wav");
+    }
+  }
+   
+#ifdef FETT263_EDIT_MODE_MENU  
+// Color Edit Helper Functions
+  void SaveColorEdit() {
+    menu_type_ = MENU_COLOR_MODE;
+    edit_color_ = false;
+    twist_menu_ = M_PI / 4;
+    NewColor(blade_num_, effect_num_); 
+    current_preset_.Save();
+    color_mode_ = NONE;
+  }
+  
+  void RevertColorEdit() {       
+    menu_type_ = MENU_COLOR_MODE;
+    edit_color_ = false;
+    twist_menu_ = M_PI / 4;
+    color_mode_ = NONE;
+  }
+#endif
+
+// Edit Mode Menu Select (PWR Button)
+  void MenuChoice() {
+    const char* tmp; 
+    switch (menu_type_) {
+      case MENU_PRESET:
+        PlayMenuSound("mselect.wav");
+        menu_type_ = MENU_TOP;
+        menu_ = false;
+        break;
+      case MENU_REHEARSE:
+        PlayMenuSound("rehrsbgn.wav");
+        rehearse_ = true;
+        MenuExit();
+        FastOn();
+        break;
+      case MENU_VOLUME:
+        if (SaberBase::IsOn()) {
+#ifdef FETT263_EDIT_MODE_MENU           
+          menu_type_ = MENU_SETTING_SUB;
+#endif           
+          MenuSave();
+        } else {
+          MenuExit();
+        }
+        break;  
+#ifdef FETT263_EDIT_MODE_MENU        
+      case MENU_TOP:
+        switch (menu_top_pos_) {
+          case EDIT_STYLE:   
+  #if NUM_BLADES == 1
+            menu_type_ = MENU_STYLE;
+            num_presets_ = current_config->num_presets;
+            blade_num_ = 1;
+            tmp = current_preset_.GetStyle(blade_num_);
+            style_num_ = FirstWord(tmp, "builtin") ? atoi(SkipWord(tmp)) : 0; 
+            style_revert_ = style_num_;               
+            PlayMenuSound("mselect.wav");
+            break;
```

Comment:
move the break after the #endif, that way you only need one (instead of two)

Reply:
fixed